sp800-63c/sec10_usability.md

```diff
@@ -3,15 +3,16 @@
 <div class="breaker"/>
```

Comment:
Note that this file also has linebreak issues, @jimfenton has the tools to fix this.